src/pages/[slug].tsx

```diff
@@ -9,24 +9,30 @@ import { getAllPagesWithSlug, getPageData, getSiteData } from "../lib/api";
 import { GridContainer } from "../components/global/grid/gridContainer";
 import { Row } from "../components/global/grid/Row";
 import { Col } from "../components/global/grid/Col";
+import PageBody from "../components/page/content";
+import styled from "@emotion/styled";
+import { getRemSize } from "../styles/globalCss";
+import { breakpoints, dimensions } from "../styles/variables";
 
 interface IProject {
   siteData: ISiteData;
   page: PageBySlug;
 }
-
+const StyledTitle = styled.h1`
+  font-size: ${getRemSize(dimensions.textSizes.xLarge.desktop)};
+  margin: 200px 0px 140px 32px;
+  @media all and (max-width: ${breakpoints.md}px) {
+    font-size: ${getRemSize(dimensions.headingSizes.display2.mobile)};
+    margin: 120px 16px;
```

Comment:
Can you remove left & right margin here 🦭
```suggestion
    margin: 120px 0;
```